src/common/options/crimson.yaml.in

```diff
@@ -2,6 +2,18 @@
 ---
 
 options:
+- name: crimson_osd_objectstore
+  type: str
+  level: advanced
+  desc: backend type for a Crimson OSD (e.g seastore or bluestore)
+  default: seastore
+  enum_values:
+  - bluestore
+  - seastore
+  - cyanstore
+  - memstore
```

Comment:
memstore should be illegal for crimson, right?

Reply:
I thought it could potentially work with Alienstore but there's probably no good reason to support that. Will remove.